tests/sparsezoo/objects/test_model_folder_objects.py

```diff
@@ -0,0 +1,358 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import pathlib
+import tempfile
+from collections import OrderedDict
+
+import numpy as np
+import onnx
+import pytest
+from onnx.helper import make_graph, make_model, make_node, make_tensor_value_info
+
+from PIL import Image
+from src.sparsezoo.objects.model_folder_objects import (
+    Directory,
+    File,
+    FrameworkFiles,
+    NumpyDirectory,
+    SampleOriginals,
+)
+from src.sparsezoo.utils.numpy import load_numpy_list, save_numpy
```

Comment:
```suggestion
from sparsezoo.objects.model_folder_objects import (
    Directory,
    File,
    FrameworkFiles,
    NumpyDirectory,
    SampleOriginals,
)
from sparsezoo.utils.numpy import load_numpy_list, save_numpy
```

Reply:
Adding it to the new PR in the other branch.